packages/app/src/state/cortex-config/sagas/index.ts

```diff
@@ -51,7 +52,7 @@ function* saveRuntimeConfig(
   action: ReturnType<typeof actions.saveCortexRuntimeConfig>
 ) {
   try {
-    yield axios.request({
+    yield request.request({
```

Comment:
We need to apply the discussion from https://github.com/opstrace/opstrace/pull/1293 to every single HTTP request.

This here is of category (3) in https://github.com/opstrace/opstrace/pull/1293#issuecomment-916184044. Should get its own set of custom retry parameters.